webapp/store/content/distros/opensuse.yaml

```diff
@@ -6,15 +6,15 @@ logo-mono: https://assets.ubuntu.com/v1/e4e63887-Distro_Logo_OpenSUSE_White.svg
 install:
   -
     action: |
-      Snap can be installed from the command line on openSUSE Leap 42.3, Leap 15 and Tumbleweed.
+      Snap can be installed from the command line on openSUSE Leap 15 and Tumbleweed.
```

Comment:
Is it not supposed to be openSUSE Leap `15.2` instead of `15` here?

Reply:
Maybe 15.x to cover all series 15 releases.

Reply:
At the moment, 15.0, 15.1, and 15.2 are all supported.
I think 15.x makes that clear.